browser-extension/addTournesolRecommendations.css

```diff
@@ -28,25 +28,83 @@
 
 .language_option {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   float: right;
   margin-top: 3px;
-  margin-left: 4px;
+  margin-left: 2px;
 }
 
 #tournesol_link {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   position: absolute;
   right: 0px;
   top: 15px;
 }
 
+#tournesol_refresh_button {
+  float: right;
+  margin-top: 4px;
+  margin-left: 12px;
+  margin-right: 16px;
+}
+
+#tournesol_expand_button {
+  float: right;
+  margin-top: 4px;
+  margin-left: 12px;
+}
+
 #tournesol_icon {
   position: absolute;
   left: 0px;
 }
 
+.tournesol_simple_button {
+  fill: var(--yt-spec-text-secondary);
+  width: 20px;
+  height: 20px;
+  padding: 0px;
+  border-style: none;
+  background-color: transparent;
+  position: relative;
+}
+
+.tournesol_simple_button svg {
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  width: 20px;
+  height: 20px;
+  border-radius: 10px;
```

Comment:
The `border-radius` here crops the icons image, see:

**with border radius** the refresh icon doesn't look good

![capture](https://user-images.githubusercontent.com/39056254/148923034-f9540dd0-64d3-4ccf-b35d-0575935c612f.png)

**without border radius** the refresh icon looks better

![capture2](https://user-images.githubusercontent.com/39056254/148923120-ab1957ca-b6b2-4653-a4dc-f07912f9a416.png)

To avoid cropping images, I recommend not using `border-radius` when there is no padding, or no background, or at least nothing to " round " : )